src/Microsoft.DotNet.Build.Tasks/PackageFiles/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "profiles": {
+        ".NET Core xUnit Console": {
+            "commandName": "Executable",
+            "executablePath": "$(BinDir)testhost/netcoreapp-$(_bc_OSGroup)-$(ConfigurationGroup)-$(ArchGroup)/dotnet.exe",
+            "commandLineArgs": "xunit.console.dll $(XunitArguments) -wait -parallel none",
+            "workingDirectory": "$(TestWorkingDir)$(MSBuildProjectName)/netcoreapp-$(_bc_OSGroup)-$(ConfigurationGroup)-$(ArchGroup)/"
+        },
+        ".NET Framework xUnit Console": {
```

Comment:
Do we want to blindly always have .NET Framework as a configuration even if the test project doesn't support it?

Reply:
For now I think that's the best way but I created https://github.com/dotnet/project-system/issues/4051 earlier today to remove the copying and be able to attach the launch profiles to certain configurations.